### ISSUE TITLE
Confusing donation (wrong amount)

### ISSUE THREAD

**Issue by LeifSimmons** [liberapay/liberapay.com#865]
I have Liberapay set to donate €0.50 per week to the Librepay team.
Today "payday" withdrew €0.46 from my wallet?!?
Very confusing!

**Comment by Changaco** [comment]
Yes it is. The amounts of donations to teams are actually treated as maximums, unlike donations to individuals and organizations which are currently processed either completely or not at all.

**Comment by LeifSimmons** [comment]
How very confusing!
I think it also leads to distrust; _"if a system transfers less than the amount I tell it to, can I trust it to not not transfer more?"_. I think you either have to fix this or at least explain it well.

**Comment by Changaco** [comment]
Related issue: #753.
